docs/components/table.md

```diff
@@ -893,18 +747,93 @@ Set the `isLoading` prop to `true` to enable the loading state.
 <KCard>
   <template v-slot:body>
     <KTable
-      :options="{}"
-      :isLoading="true"
-    />
+      :fetcher="fetcher"
+      :headers="headers"
+      :isLoading="true" />
   </template>
 </KCard>
 </template>
 ```
 
+## Server-side functions
+
+Pass a fetcher function to enable server-side search, sort and pagination.
+The fetcher function should structure the ajax request URL in such a way that
+enables server side sort, search and pagination per the requirements of the API
+being used.
+
+<KTable
+  :fetcher="fetcher"
+  :headers="headers" />
+
+```http
+Example URL
+
+https://kongponents.dev/api/components?_page=1&_limit=10&_sort=name&_order=desc
+```
+
+```vue
+<!-- Example Component Usage -->
+
+<KCard>
+  <template v-slot:body>
+    <KInput placeholder="Search..." v-model="search" type="search" />
+    <KTable
+      :fetcher="fetcher"
+      :initial-fetcher-params="{
+        pageSize: 15,
+        page: 1,
+        query: '',
+        sortColumnKey: '',
+        sortColumnOrder: ''
+      }"
+      :headers="[
+        { label: 'Title', key: 'title', sortable: true },
+        { label: 'Description', key: 'description', sortable: true },
+        { label: 'Enabled', key: 'enabled', sortable: false }
+      ]"
+      :search-input="search"
+    />
+  </template>
+</KCard>
+```
+
+```js
+// Example Fetcher Function
+
+fetcher(payload) {
```

Comment:
suggestion: Reference that loading state is taken care of internally.

Reply:
Done